FREEDOM_COVENANT.txt

```diff
@@ -255,7 +255,7 @@ Nobody has any right whatsoever to threaten another individual or to demand
 retribution against them. Such demands should be politely ignored or laughed
 at, because they are direct attacks on freedom itself.
 
-Sharing facts requires us to acknowledge that such attacks are been made, on an
```

Comment:
This part of the text was reworded. Someone else pointed out that it was a bit too edgy.